conformance/operator_test.go

```diff
@@ -466,6 +466,57 @@ var _ = Describe("operator", func() {
 				validateNetworkFields(copyObj, trustChkStatusValidation)
 			})
 
+			// 25961
+			It("Should configure the the link state variable", func() {
+				sriovNetwork := &sriovv1.SriovNetwork{
+					ObjectMeta: metav1.ObjectMeta{Name: "statenetwork", Namespace: operatorNamespace},
+					Spec: sriovv1.SriovNetworkSpec{
+						ResourceName: "testresource",
+						IPAM: `{"type":"host-local",
+									"subnet":"10.10.10.0/24",
+									"rangeStart":"10.10.10.171",
+									"rangeEnd":"10.10.10.181",
+									"routes":[{"dst":"0.0.0.0/0"}],
+									"gateway":"10.10.10.1"}`,
+						NetworkNamespace: namespaces.Test,
+					}}
+
+				By("configuring link-state enable")
+				copyObj := sriovNetwork.DeepCopy()
+				copyObj.Spec.LinkState = "enable"
+				linkStateChkStatusValidation := "link-state enable"
+				err := clients.Create(context.Background(), copyObj)
+				Expect(err).ToNot(HaveOccurred())
+
+				validateNetworkFields(copyObj, linkStateChkStatusValidation)
+
+				By("removing sriov network")
+				err = clients.Delete(context.Background(), sriovNetwork)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("configuring link-state disable")
+				copyObj = sriovNetwork.DeepCopy()
+				copyObj.Spec.LinkState = "disable"
+				linkStateChkStatusValidation = "link-state disable"
+				err = clients.Create(context.Background(), copyObj)
+				Expect(err).ToNot(HaveOccurred())
+
+				validateNetworkFields(copyObj, linkStateChkStatusValidation)
+
+				By("removing sriov network")
+				err = clients.Delete(context.Background(), sriovNetwork)
```

Comment:
Can you use the one you just created for deletion? I think makes things a bit more clear.